packages/web3/test/black_box/test/cjs_tests/batchRequest.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { getSystemTestProvider, closeOpenConnection, isWs } from '../fixtures/system_tests_utils';
```

Comment:
cjs_tests and esm_tests should both be different by having their own package.json and :

`"type": "module"` for esm blackbox in package json , 
`"type": "commonjs"` for cjs blackbox package.json

also each should have its own jest-config and `ts-jest` extending to respective cjs and esm tsconfig files. Like https://github.com/web3/web3.js/blob/8da5d8331e18271d573f498fb046aa87fc096dd2/packages/web3/test/config/jest.config.js#L4 is extending ./test/tsconfig.json , that is extending ../tsconfig.cjs.json

Reply:
updated